catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -433,14 +442,18 @@ public void init() {
         });
 
         exception(RuntimeException.class, (ex, req, res) -> {
-            LOGGER.debug("Exception occured.", ex);
+            LOGGER.debug("Exception occurred.", ex);
             res.status(404);
             res.header(CONTENT_TYPE, APPLICATION_JSON);
             res.body(util.getJson(ImmutableMap.of("message",
                     "Could not find what you were looking for")));
         });
     }
 
+    private String getJsonWritableSources() {
+        return util.getJson(getWritableSources());
```

Comment:
just a nit which you can fix or ignore: i would just inline this since its only 1 line and not even a very long line.